src/lib/metricsMgr/MetricsManager.h

```diff
@@ -41,28 +94,33 @@
 *    initial map created at constructor time)
 * 3. Destroy method, releasing all the maps in cascade (probably never used, as the
 *    singleton object in CB using this class will be destroyed at the end, but do it
-*    for class compleness)
+*    for class completeness)
 * 4. reset() method implementation (not delete maps, only set metrics to 0)
-* 5. toJson() to be splitted into 3 methods (2 of them private)
+* 5. toJson() to be split into 3 methods (2 of them private)
```

Comment:
Probably step 6 can be removed from this PR (as you have added the sem management logic in the different methods).